build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-dokka.gradle.kts

```diff
@@ -21,13 +21,13 @@ tasks.withType<AbstractDokkaTask>().configureEach {
     )
 }
 
-tasks.withType<DokkaTaskPartial>().configureEach {
+dokka {
```

Comment:
configuration from above should be also migrated to `dokka` extension.
e.g something like this:
```
dokka {
  pluginsConfiguration.html {
    templatesDir = rootDir.resolve("dokka-templates")
  }
}
```

Reply:
Thanks!